views/partials/nav.hbs

```diff
@@ -1,5 +1,7 @@
 <link rel="stylesheet" href="/css/navbar.css">
 <script src="/js/app.js" async></script>
```

Comment:
Ito yung file drop feature sa file upload dati. I'll remove it so we can stick to a simple file upload.

Reply:
Noted.